packages/ibm-products-web-components/src/components/options-tile/options-tile.mdx

```diff
@@ -6,7 +6,15 @@ import * as OptionsTileStories from './options-tile.stories';
 
 # OptionsTile
 
-The OptionsTile component TODO
+> 💡 Check our
+> [Stackblitz](#)
+> example implementation.
+
+[![Edit carbon-web-components](https://developer.stackblitz.com/img/open_in_stackblitz.svg)](https://stackblitz.com/github/carbon-design-system/ibm-products/tree/main/packages/ibm-products-web-components/examples/options-tile)
+
+An options tile can contain information, controls or tables which, when collapsed, are summarized. It can be paired with a toggle to quickly enable or disable the option.
+
+[Options Tile guidelines (IBM internal)](https://stackblitz.com/github/carbon-design-system/ibm-products/tree/main/packages/ibm-products-web-components/examples/options-tile)
```

Comment:
I realize we include this in the React docs too but think we can remove for now until we decide on a standard to include in all our the WC docs pages going forward.

```suggestion
```